handler/cantabular_metadata_export_test.go

```diff
@@ -33,12 +33,17 @@ func TestGenerateTextFilename(t *testing.T) {
 
 func TestGenerateVaultPath(t *testing.T) {
 
-	Convey("Succesfully return a string representing the vault path", t, func() {
+	Convey("Succesfully return a string representing the vault path with given file extension", t, func() {
 		cfg, err := config.Get()
 		So(err, ShouldBeNil)
 		h := NewCantabularMetadataExport(*cfg, nil, nil, nil)
-		vaultPath := h.generateVaultPath(&eventTest)
-		expectedVaultPath := "secret/shared/psk/test_id-test-edition-1.txt"
+
+		vaultPath := h.generateVaultPath(&eventTest, "foo")
+		expectedVaultPath := "secret/shared/psk/test_id-test-edition-1.foo"
+		So(vaultPath, ShouldResemble, expectedVaultPath)
+
+		vaultPath = h.generateVaultPath(&eventTest, "bar")
```

Comment:
Should we consider creating additional `Conveys` for these cases? I wouldn't expect two calls to `generateVaultPath` in the same test case